sdc/tests/test_utils.py

```diff
@@ -94,21 +94,11 @@ def gen_df_int_cols(input_data, with_index=False):
 
 
 def count_array_REPs():
-    if sdc.config.config_pipeline_hpat_default:
-        from sdc.distributed import Distribution
-        vals = sdc.distributed.dist_analysis.array_dists.values()
-        return sum([v == Distribution.REP for v in vals])
-    else:
-        return 0
+    return 0
 
 
 def count_parfor_REPs():
```

Comment:
These functions (count_array_REPs and count_parfor_REPs) were used in old pipeline only anyway. So if we keep them, it's better remove the other branch (with return 0) and keep impl from sdc.config.config_pipeline_hpat_default == True branch (for reference only).